### DISCUSSION TITLE
Any plans to add other loss functions?

### DISCUSSION THREAD

**Discussion by dosp0911** [pyg-team/pytorch_geometric#2324] [Q&A] [answered]
Hi. 
I was wondering if you have a plan to add some loss functions for generating 3d data ? like chamfer loss, edge_loss, laplacian loss. etc? I'm working on Graph Auto Encoder for mesh data. but I couldn't find the loss function. Thanks

**Comment by rusty1s** [comment, marked as answer]
No plans yet, but those could be indeed be very useful. Please feel free to contribute them :)
